.github/workflows/build.yml

```diff
@@ -9,95 +9,61 @@ name: Continuous integration
-          args: --release --all
-      
-      - name: Upload Artifact (Linux/MacOS)
-        uses: actions/upload-artifact@v1
-        if: matrix.rust == 'stable' && matrix.os != 'windows-latest'
-        with:
-          name: juno-${{matrix.os}}
-          path: target/release/juno
-      
-      - name: Upload Artifact (Windows)
-        uses: actions/upload-artifact@v1
-        if: matrix.rust == 'stable' && matrix.os == 'windows-latest'
-        with:
-          name: juno-${{matrix.os}}
-          path: target/release/juno.exe
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Linting and Test
+        run: npm run test:prod
+
+      - name: Run Build
+        run: npm run build
 
-  release-master: # Publish release on push to master
+  # Publish release on push to master
+  release-master:
     if: github.ref == 'refs/heads/master'
     runs-on: ubuntu-latest
     needs: build
     steps:
       - uses: actions/checkout@v2
+
       - run: git fetch --all --tags
 
       - name: Check Release Version
         uses: thebongy/version-check@v1
         with:
-          file: Cargo.toml
+          file: package.json
           tagFormat: v${version}
         id: version_check
-      
-      - name: Download Windows Artifact
-        uses: actions/download-artifact@v1
-        with:
-          name: juno-windows-latest
-          path: release/windows
-
-      - name: Download MacOS Artifact
-        uses: actions/download-artifact@v1
-        with:
-          name: juno-macos-latest
-          path: release/macos
-
-      - name: Download Linux Artifact
-        uses: actions/download-artifact@v1
-        with:
-          name: juno-ubuntu-latest
-          path: release/linux
-      
-      - name: Rename Artifacts
-        run: |
-          mv release/windows/juno.exe release/windows/juno-${{steps.version_check.outputs.releaseVersion}}-windows.exe
-          mv release/macos/juno release/macos/juno-${{steps.version_check.outputs.releaseVersion}}-macos
-          mv release/linux/juno release/linux/juno-${{steps.version_check.outputs.releaseVersion}}-linux
 
       - name: Publish Release
-        uses: softprops/action-gh-release@v1
+        id: create_release
+        uses: actions/create-release@latest
         with:
-          files: release/**/*
-          tag_name: ${{steps.version_check.outputs.releaseVersion}}
+          tag_name: ${{ steps.version_check.outputs.releaseVersion }}
+          release_name: Juno-node @Linux Release ${{ steps.version_check.outputs.releaseVersion }}
           prerelease: false
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Upload Release Asset
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: ./juno-node-linux-${{ steps.version_check.outputs.releaseVersion }}.zip
```

Comment:
This asset doesn't exist in this job. Actions don't persist files b/w jobs. You'll have to run `npm pack` in the build job,  and then use `actions/upload-artifact@v1` to upload the artifact (the tar file generated by npm pack) there, and then `actions/download-artifact@v1` to download that artifact back into this job. Then you can upload that download tar file as an asset in this job.